src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/ITypeSymbolExtensions.TypeSyntaxGeneratorVisitor.cs

```diff
@@ -127,18 +127,56 @@ public static bool TryCreateNativeIntegerType(INamedTypeSymbol symbol, out TypeS
 
 #if !CODE_STYLE
 
+#nullable enable
             public override TypeSyntax VisitFunctionPointerType(IFunctionPointerTypeSymbol symbol)
             {
-                // TODO(https://github.com/dotnet/roslyn/issues/39865): generate the calling convention once exposed through the API
+                FunctionPointerCallingConventionSyntax? callingConventionSyntax = null;
```

Comment:
Move declaration near assignment -- nothing is ever seeing the 'null' since you overwrite it on line 160.

Reply:
It's observed if the `if` below isn't taken.

---
In reply to: [463857030](https://github.com/dotnet/roslyn/pull/46467#discussion_r463857030) [](ancestors = 463857030)